### PR TITLE
[15.0][MIG] purchase_location_by_line

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Allows to define a specific destination location on each Purchase Order line.
When the PO is confirmed, it will generate one Incoming Shipment per
combination of destination location and expected date indicated in the
Purchase Order Lines.

This module is used to send the products to different locations, that may
not be children of the default location of the same PO picking type. Not be confused.

### CODE REVIEW

File: purchase_location_by_line/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# © 2016 ForgeFlow S.L.
+#   (<http://www.forgeflow.com>)
+# © 2018 Hizbul Bahar <hizbul25@gmail.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Purchase Location by Line",
+    "summary": "Allows to define a specific destination location on each PO line",
+    "version": "15.0.1.0.1",
```
Comment (MiquelRForgeFlow) [inline purchase_location_by_line/__manifest__.py]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by HviorForgeFlow — COMMENTED** [review]
Functional review.

It is possible to change Destination Location once Purchase order is confirm (equal to picking is also created). I guess blocking the capacity to edit the field when ['purchase', 'done', 'cancelled'] could be a solution.

**Review by HviorForgeFlow — APPROVED** [review]
Functional Review :+1: 
Thanks for the improvement

**Pull request merged**